docs/features/techdocs/using-cloud-storage.md

```diff
@@ -195,3 +195,68 @@ Your Backstage app is now ready to use AWS S3 for TechDocs, to store and read
 the static generated documentation files. When you start the backend of the app,
 you should be able to see
 `techdocs info Successfully connected to the AWS S3 bucket` in the logs.
+
+## Configuring Azure Blob Storage Container with TechDocs
+
+Follow the
+[official Azure Blob Storage documentation](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth?toc=/azure/storage/blobs/toc.json)
+for the latest instructions on the following steps involving Azure Blob Storage.
+
+**1. Set `techdocs.publisher.type` config in your `app-config.yaml`**
+
+Set `techdocs.publisher.type` to `'azureBlobStorage'`.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'azureBlobStorage'
+```
+
+**2. Service account credentials**
+
+To get credentials, access the Azure Portal and go to "Settings > Access Keys",
+and get your Storage account name and Primary Key.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'azureBlobStorage'
+    azureBlobStorage:
+      credentials:
+        account:
+          $env: TECHDOCS_AZURE_BLOB_STORAGE_ACCOUNT
+        accountKey:
+          $env: TECHDOCS_AZURE_BLOB_STORAGE_ACCOUNT_KEY
+```
+
+**3. Azure Blob Storage Container**
+
+Create a dedicated container for TechDocs sites. techdocs-backend will publish
+documentation to this container. TechDocs will fetch files from here to serve
+documentation in Backstage.
+
+To create a new container, access "Blob Service > Containers > New Container".
+
+Set the name of the container to
+`techdocs.publisher.azureBlobStorage.containerName`.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'azureBlobStorage'
+    azureBlobStorage:
+      containerName: 'name-of-techdocs-storage-container'
+      credentials:
+        account:
+          $env: TECHDOCS_AZURE_BLOB_STORAGE_ACCOUNT
+        accountKey:
+          $env: TECHDOCS_AZURE_BLOB_STORAGE_ACCOUNT_KEY
+```
```

Comment:
Let's move the 3. container settings to number 2. and after that, let's create "3a (recommended)" which has the instructions about using environment variables to authenticate, and 3b. which suggests the settings credentials via `app-config.yaml`. Do take a look at AWS/GCS for inspiration https://backstage.io/docs/features/techdocs/using-cloud-storage